File: Ohtap E Commerce Domain Models/Development/PHP/Objects/PackageOptions/IPackageOptionData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\PackageOptions;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;



interface IPackageOptionData extends IPackageOptionOption,IPackageOptionValue,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/PackageOptions/IPackageOptionData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.PackageOptions
{
    public interface IPackageOptionData :  CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IPackageOptionValue, IPackageOptionOption
    {

    }
}
